src/igvfd/types/file_set.py

```diff
@@ -41,6 +42,18 @@ class FileSet(Item):
     def files(self, request, files):
         return paths_filtered_by_status(request, files)
 
+    @calculated_property(schema={
+        'title': 'MeasurementSets this FileSets act as a control.',
+        'type': 'array',
+        'items': {
+            'type': ['string', 'object'],
+            'linkFrom': 'FileSet.control_file_sets',
+        },
+        'notSubmittable': True
+    })
+    def controlling_file_sets(self, request, controlling_file_sets):
+        return paths_filtered_by_status(request, controlling_file_sets)
```

Comment:
title: File sets controlled by this file set. 
rev-link name: "controlled_file_sets"

Reply:
rev-link name: control_of
fixed